src/Api/Api.ts

```diff
@@ -0,0 +1,15 @@
+const TODOS_API = 'https://jsonplaceholder.typicode.com/todos';
+const USERS_API = 'https://jsonplaceholder.typicode.com/users';
```

Comment:
если нам нужно будет много чего получать с этого апи, будем создавать новые константы?

```suggestion
const API_URL = 'https://jsonplaceholder.typicode.com/';
```
`todos` `users` дописывай по ситуации